website/source/api/jobs.html.md

```diff
@@ -972,9 +972,9 @@ The table below shows this endpoint's support for
 [blocking queries](/api/index.html#blocking-queries) and
 [required ACLs](/api/index.html#acls).
 
-| Blocking Queries | ACL Required |
-| ---------------- | ------------ |
-| `YES`            | `none`       |
+| Blocking Queries | ACL Required               |
+| ---------------- | -------------------------- |
```

Comment:
Extra width?

Reply:
The line below adds more width so this is to bring all lines to the same width.